doc/getting-started/Kotlin-Versions.md

```diff
@@ -0,0 +1,29 @@
+## Kotlin versions and the Gradle Kotlin DSL 
+
+Gradle ships with Kotlin DSL support which uses embedded Kotlin compiler, plus fixed Kotlin `stdlib` and `reflect`
```

Comment:
⭕️  I would rephrase the first sentence slightly to:

> Gradle Kotlin DSL ships with the embedded Kotlin compiler plus matching versions of the Kotlin `stdlib` and `reflect` libraries.

Reply:
I like it better 👍 
Done